Zend/tests/lc_ctype_inheritance.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Do not inherit LC_CTYPE from environment
+--SKIPIF--
+<?php
+if (!setlocale(LC_CTYPE, "de_DE")) die("skip requires de_DE locale");
```

Comment:
That would always skip the test on Windows, which wants a dash instead of an underscore.
```suggestion
if (!setlocale(LC_CTYPE, "de_DE", "de-DE")) die("skip requires de_DE locale");
```